### ISSUE TITLE
Unknown error from enum Rule

### ISSUE THREAD

**Issue by wolffan** [realm/SwiftLint#1542]
When I lint my project I get the following:
`Enum element name should only contain alphanumeric characters: '<enumName>'

But I can't find the string related to this error on the project github :/.
I'm on version 0.18.1 (I think it's the last one)

**Comment by wolffan** [comment]
```
disabled_rules:
  - colon
  - comma
  - control_statement
  - 
opt_in_rules:
  - force_unwrapping
  - overridden_super_call
  - prohibited_super_call
  - discarded_notification_center_observer
  - dynamic_inline
  - first_where
  - implicitly_unwrapped_optional
  - valid_ibinspectable
  - weak_delegate
  - file_length

included: # paths to include during linting. `--path` is ignored if present.
  - RocketCore
  - RocketApp
  - RocketUI
excluded: # paths to ignore during linting. Takes precedence over `included`.
  - Pods

force_try:
  severity: warning

function_body_length:
  warning: 100
  error: 200
line_length: 160
reporter: "checkstyle" # reporter type (xcode, json, csv, checkstyle, junit, html, emoji)
```
Here's the swiftlint.yml

**Comment by marcelofabri** [comment]
This is triggered by the `identifier_name` rule: https://github.com/realm/SwiftLint/blob/61e9b39327b22b237dd06c4254f339b881a44e1f/Source/SwiftLintFramework/Rules/IdentifierNameRule.swift#L57

**Comment by wolffan** [comment]
👍  🤦‍♂️ 
Thanks @marcelofabri  and thanks!

**Comment by marcelofabri** [comment]
No worries! Feel free to reopen or open another issue if you have more questions 😊
